javascriptv3/example_code/cross-services/textract-react/README.md

```diff
@@ -0,0 +1,211 @@
+  on the AWS website.
+- This example uses React and Bootstrap, popular libraries for building user interfaces 
+that run in a web browser. Use of these libraries is for illustrative purposes only 
+and does not indicate endorsement.
+- Running this code might result in charges to your AWS account.
+
+## Prerequisites and setup
+
+1. You must have an AWS account and have your default credentials and AWS Region
+configured as described in the [AWS Tools and SDKs Shared Configuration and
+Credentials Reference Guide](https://docs.aws.amazon.com/credref/latest/refdocs/creds-config-files.html).
+
+1. Install the dependencies listed in `package.json`. This package was tested with
+npm version 6.14.9.
+
+    ```
+    cd javascriptv3/example_code/cross-services/textract-react 
+    npm install
+    ```
+
+1. Create AWS resources needed for the example by running the `setup.yaml` AWS
+CloudFormation script. Substitute your own stack name for `<stack-name>` in the command.
+This name must be unique within your AWS account. 
+
+    ```
+    aws cloudformation create-stack --stack-name <stack-name> --template-body file://setup.yaml --capabilities CAPABILITY_IAM
+    ```
+
+    The `setup.yaml` file was built from the 
+    [AWS Cloud Development Kit (AWS CDK)](https://docs.aws.amazon.com/cdk/) 
+    source script here: 
+    [/resources/cdk/textract_example_s3_sns_sqs_cognito/setup.ts](https://github.com/awsdocs/aws-doc-sdk-examples/blob/master/resources/cdk/textract_example_s3_sns_sqs_cognito/setup.ts). 
+
+1. Verify the stack is completely deployed before the next step by running the 
+following command:
+
+    ```
+    aws cloudformation describe-stacks --stack-name <stack-name>
+    ```
+    
+    The stack is deployed when it reports a StackStatus of CREATE_COMPLETE.
+
+1. Set up additional items needed for the example by running the `DeployDestroy.js` 
+script with the `deploy` option and pass the stack name you used in the previous step.
+
+    ```
+    node DeployDestroy.js deploy <stack-name>
+    ```
+    
+    This script puts output data from the CloudFormation stack into the `Config.js` file 
+    and uploads a default image to Amazon S3. 
+    
+## Running the code
+
+The application runs in a web browser.
+
+1. Start the development web server by running the following commands.
+
+    ```
+    cd src
+    npm start
+    ```
+
+1. If your browser does not open automatically, open a browser and navigate to 
+[http://localhost:3000](http://localhost:3000).
+
+1. Select **Sign in**. This navigates you to an Amazon Cognito hosted UI where you
+can sign in or sign up to create an Amazon Cognito identity. If you create a new
+identity, you must provide a valid email address to receive a verification code. An
+Amazon Cognito authenticated role is required to access the asynchronous Amazon
+Textract APIs used by this example.
+
+1. After you sign in to Amazon Cognito, you are redirected back to the example 
```

Comment:
Should these step numbers increment? Maybe just a display issue?

Reply:
Markdown will auto-increment a numbered list when it renders.